docs/Reference/README.md

```diff
@@ -0,0 +1,53 @@
+# Reference Documentation
+
+How to use the Cosmos Cash Reference Documentation.
+
+[TOC]
+
+
+
+## Context
+
+In Cosmos Cash we use the  *Grand Unified Theory of Documentation* (David Laing) as described by [Divio](https://documentation.divio.com/) as a basis for our documentation strategy.
+
+This approach outlines four specific use cases for documentation:
+
+* [Tutorials](../Tutorials/README.md)
+* [How-Tos](../How-To/README.md)
+* [Explanation](./Explanation/README.md)
+* [Reference](./README.md)
+
+This section addresses the last of these: **Reference documentation**. 
+
+For further background please see [the ADR relating to the documentation structure](./ADR/adr-002-docs-structure.md). 
+
+## Approach
+
+As outlined in the strategy [Reference Documentation](https://documentation.divio.com/reference/) should be **information oriented**. It should allow the reader to easily navigate the content and use in conjunction with the code.
+
+This documentation describes the machinery, for example, classes, functions, interfaces, parameters, and so on.
+
+
+## Contributing
+
+* The content should be dry, clear and terse in style.
+* All documentation should be written following [Google Documentation Best Practice](https://google.github.io/styleguide/docguide/best_practices.html)
+* Generate as much documentation as possible from the code.
+* Raise a PR for all documentation changes
+
+## Layout
+
+Reference Documentation will come in various forms:
+
+* [Architecture diagrams](./architecture) - this will be in SVG format so that it can be stored in GitHub and version controlled. 
```

Comment:
```suggestion
* [Architecture diagrams](./architecture) - Diagrams must be in SVG format so that the diagrams can remain crisp and clear at any resolution or size, stored in GitHub, and version controlled. 
```

Reply:
we use png in our docs, so the SVG choice is probably based on: A vector image remains crisp and clear at any resolution or size. If so, let's add that reason